best-practices/README.md

```diff
@@ -264,6 +264,21 @@ Objective-C
 * Prefer categories on `Foundation` classes to helper methods.
 * Prefer string constants to literals when providing keys or key paths to methods.
 
+Android
+-------
+
+* Each callback on an `Activity` should invoke methods on an instance of a
+  presenter object.
+* Unit test each presenter object.
+* Integration tests are optional and not encouraged.
```

Comment:
I feel weird posting this publicly. I think I, Amanda, have reasons not to use integration tests and a lot of those are specific to my experiences and the fact that we do client work. I think integration tests are hard to set up and maintain especially given our tight timelines and how quickly and frequently client's change designs. But I don't think this statement is true for ALL android projects. I think at product companies they might make a great deal of sense especially when they have the time and resources for it